roles/hypervisor/templates/installserver.xml.tmpl

```diff
@@ -1,11 +1,11 @@
 <domain type='kvm'>
   <name>installserver</name>
   <uuid>2acfbe60-5eca-11e4-9803-0800200c9a69</uuid>
-  <memory unit='KiB'>2097152</memory>
-  <currentMemory unit='KiB'>2097152</currentMemory>
+  <memory unit='KiB'>{{ vm_default_memory | default(4194304) }}</memory>
+  <currentMemory unit='KiB'>{{ vm_default_memory | default(4194304) }}</currentMemory>
   <vcpu placement='static'>2</vcpu>
   <os>
-    <type arch='x86_64' machine='rhel6.6.0'>hvm</type>
+    <type arch='x86_64' machine='pc-1.2'>hvm</type>
```

Comment:
This flag does not exist on my rhel7 box, but `pc` works just fine. Is that ok on your Debian lab ?